modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/model/impl/CPDefinitionImpl.java

```diff
@@ -219,22 +218,9 @@ public CPTaxCategory getCPTaxCategory() throws PortalException {
 	}
 
 	@Override
-	public String getDefaultImageFileURL() throws PortalException {
```

Comment:
Has this method a replacement?

Reply:
Or it is just not used anymore?

Reply:
Yes, I will deprecate it in the past versions

Reply:
I moved the logic into the cpDefinitionHelper